app/services/storage/s3_service.rb

```diff
@@ -20,10 +20,10 @@ def folder_present?(folder)
       response.key_count == 1
     end
 
-    def get_presigned_url(file_name, duration)
+    def get_presigned_url(file_name, duration, response_content_disposition: nil)
       Aws::S3::Presigner
         .new({ client: @client })
-        .presigned_url(:get_object, bucket: @configuration.bucket_name, key: file_name, expires_in: duration)
+        .presigned_url(:get_object, bucket: @configuration.bucket_name, key: file_name, expires_in: duration, response_content_disposition:)
```

Comment:
if response_content_disposition is nil do we override a default?

Reply:
The issue stemmed from when downloading a bulk upload it would've come up as an blank file without specifying response_content_disposition, I did set the method to initialise with nil unless specified. This did not affect downloading other CSVs elsewhere in the app. This could be done better though, so will look at it again.